docs/chain-details/module_overview.md

```diff
@@ -0,0 +1,13 @@
+# Modules - overview
+
```

Comment:
Devashish recently added "module" for the supply querying, so perhaps good to document that too?

Reply:
Sure, I have included the supply module in https://github.com/crypto-com/chain-docs/pull/38/commits/b9e6333043961f87d336f7245182d9d4c38e407a.